.gitlab/pipelines/stages/deploy/wait_kubecf.sh

```diff
@@ -9,51 +9,38 @@ source "${workspace}/.gitlab/pipelines/runtime/config.sh"
 # shellcheck disable=SC1090
 source "${workspace}/.gitlab/pipelines/runtime/binaries.sh"
 
-wait_kubecf() {
-  timeout 900 bash <<EOF
-while true; do
-  if '${KUBECTL}' get pods --namespace ${KUBECF_NAMESPACE} --no-headers --ignore-not-found | grep "router"; then
-    exit 0
-  fi
+echo "Waiting for the kubecf pods to be ready..."
+
+secret_name="${KUBECF_INSTALL_NAME}.with-ops"
+
+until "${KUBECTL}" get secret "${secret_name}" \
+  --namespace "${KUBECF_NAMESPACE}" \
+  --output jsonpath='{ .data.manifest\.yaml }' \
```

Comment:
Since we're not reading it here, we can `--output=name`, no?

I wonder if we should document that this wait script only works for initial deployment, not upgrades / config changes…

Reply:
I wanted to make sure the secret exists with that key inside, not just the secret exists.